src/components/standup/index.tsx

```diff
@@ -0,0 +1,135 @@
+import { FC, useState, useEffect } from 'react';
+import moment from 'moment';
+import styles from '@/components/standup/standupContainer.module.scss';
+
+import {
+    useSaveProgressMutation,
+    useUserProgressDetailsQuery,
+} from '@/app/services/progressesApi';
+import { useGetUserQuery } from '@/app/services/userApi';
+
+import FormInputComponent from './FormInputComponent';
+import { standupUpdateType } from '@/types/standup.type';
+import { getTotalMissedUpdates } from '@/utils/getTotalMissedUpdate';
+import { toast, ToastTypes } from '@/helperFunctions/toast';
+import {
+    ERROR_MESSAGE,
+    STANDUP_SUBMISSION_SUCCESS,
+} from '@/constants/constants';
+import ProgressHeader from '../ProgressForm/ProgressHeader';
+
+const defaultState = {
+    type: 'user',
+    completed: '',
+    planned: '',
+    blockers: '',
+};
+
+const StandUpContainer: FC = () => {
+    const [standupUpdate, setStandupUpdate] =
+        useState<standupUpdateType>(defaultState);
+
+    const [addStandup] = useSaveProgressMutation();
+    const { data: user } = useGetUserQuery();
+    const { data: userStandupdata } = useUserProgressDetailsQuery(user?.id);
+
+    const { SUCCESS, ERROR } = ToastTypes;
+    const standupDates = userStandupdata?.data?.map((element) => element.date);
+    const totalMissedUpdates = getTotalMissedUpdates(standupDates || []);
+    const yesterdayDate = moment().subtract(1, 'days').format('MMMM DD, YYYY');
+
+    const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setStandupUpdate((prevStandupUpdate) => ({
+            ...prevStandupUpdate,
+            [event.target.name]: event.target.value,
+        }));
+    };
+    const isValidate = () => {
+        return (
+            standupUpdate.completed !== '' &&
+            standupUpdate.planned !== '' &&
+            standupUpdate.blockers !== ''
+        );
+    };
+
+    const buttonStyleClass = !isValidate()
+        ? `${styles.nonActiveButton}`
+        : `${styles.activeButton}`;
+
+    const handleFormSubmission = async (
+        event: React.FormEvent<HTMLFormElement>
+    ) => {
+        event.preventDefault();
+        try {
+            await addStandup(standupUpdate);
+            toast(SUCCESS, STANDUP_SUBMISSION_SUCCESS);
+            setStandupUpdate(defaultState);
+        } catch (error) {
+            console.error(error);
```

Comment:
do we need console error in frontend?

Reply:
https://github.com/Real-Dev-Squad/website-status/pull/518#discussion_r1212590937

Vaibhav asked me to put console.error